BUILD.gn

```diff
@@ -977,6 +978,11 @@ if (is_mac) {
         deps += [ "//sandbox/mac:seatbelt" ]
       }
       defines = [ "HELPER_EXECUTABLE" ]
+      if (is_mas_build) {
+        defines += [ "IS_MAS_BUILD()=1" ]
+      } else {
+        defines += [ "IS_MAS_BUILD()=0" ]
+      }
```

Comment:
how come these are duplicated from the config?

Reply:
i don't think that's true? https://source.chromium.org/chromium/chromium/src/+/main:build/config/mac/rules.gni;l=479-498;drc=88d2dd9aa6fe5071c862699b53aab45bd8a5ad93

either `configs` or `extra_configs` should work.

Reply:
Oh hm `extra_configs`, I didn't try that one lol